src/pages/Categories.jsx

```diff
@@ -0,0 +1,12 @@
+import Button from '../components/button/Button';
+import './pages.scss';
+
+const CategoriesScreen = () => (
+  <div className="screen">
+    <div className="container text-center">
+      <Button label="CHECK STATUS" size="big" axn={() => console.log('check status btn pressed')} />
```

Comment:
- [ ] [OPTIONAL] Well for the next time try to avoid using `coneole.log` because it's too dangerous line of code.